From 699c8ff6538644f1f198b55189f23036ac39b61d Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Fri, 14 Aug 2020 10:58:04 +0800
Subject: [PATCH] drm/rockchip: init unplug in connector status

As like TVE connector, it's unsupported hot plug, so the connector status is
always connected, this will lead to print waring log when show kernel logo.

Change-Id: Ie1b0ff1630e5f5f425844b550cd7c88cd26c29d8
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_drm_drv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
index 8fa6546825e5..7e944228e760 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
@@ -694,6 +694,7 @@ static int setup_initial_state(struct drm_device *drm_dev,
 
 	if (!found) {
 		ret = -EINVAL;
+		connector->status = connector_status_disconnected;
 		goto error_conn;
 	}
 
-- 
2.35.3

